packages/sanity/src/core/search/text-search/createTextSearch.test.ts

```diff
@@ -0,0 +1,188 @@
+import {describe, expect, it} from '@jest/globals'
+import {Schema} from '@sanity/schema'
+import {defineField, defineType} from '@sanity/types'
+
+import {getDocumentTypeConfiguration, getSort} from './createTextSearch'
```

Comment:
It looks like maybe there was a regression here and these functions are no longer exported.

I'm guessing this is getting resolved in all the force pushing I'm seeing in this branch, but just making a note.

Reply:
yes nice catch. this is fixed!